_posts/2024-04-18-geoserver-2-24-3-released.md

```diff
@@ -0,0 +1,107 @@
+---
+author: Andrea Aime
+date: 2024-04-18
+layout: post
+title: GeoServer 2.24.3 Release
+categories:
+- Announcements
+tags:
+- Release
+release: release_224
+version: 2.24.3
+jira_version: 16912
+--- 
+
+GeoServer [2.24.3](/release/2.24.3/) release is now available
+with downloads
+([bin](https://sourceforge.net/projects/geoserver/files/GeoServer/2.24.3/geoserver-2.24.3-bin.zip/download),
+[war](https://sourceforge.net/projects/geoserver/files/GeoServer/2.24.3/geoserver-2.24.3-war.zip/download),
+[windows](https://sourceforge.net/projects/geoserver/files/GeoServer/2.24.3/GeoServer-2.24.3-winsetup.exe/download)), along with 
+[docs](https://sourceforge.net/projects/geoserver/files/GeoServer/2.24.3/geoserver-2.24.3-htmldoc.zip/download) and
+[extensions](https://sourceforge.net/projects/geoserver/files/GeoServer/2.24.3/extensions/).
+
+This is a maintenance release of GeoServer providing existing installations with minor updates and bug fixes.
+GeoServer 2.24.3 is made in conjunction with GeoTools 30.3, and GeoWebCache 1.24.3. 
+
+Thanks to Andrea Aime for making this release. 
```

Comment:
Do you want to acknowledge your employer here, or was this done in your personal capacity?

Reply:
Well this was a surprise... I noticed it too and fixed it before pushing... and now it's not there?
Turns out I did not save in VS Code, and the edit remained in memory (funny enough, VS Code preserved the edit across restarts, too). Thanks, really fixed now.